src/main/java/com/esri/samples/geometry/cut_geometry/README.md

```diff
@@ -0,0 +1,38 @@
+<h1>Cut Geometry</h1>
+
+<p>Demonstrates how to cut a geometry with a polyline using the GeometryEngine.</p>
+
+<p><img src="CutGeometry.png"/></p>
+
+<h2>How to use the sample</h2>
+
+<p>Click the "Cut" button to cut the polygon with the polyline and see the resulting parts.</p>
+
+<h2>How it works</h2>
+
+<p>To cut a geometry with a polyline:</p>
+
+<ol>
+    <li>Use the static method <code>GeometryEngine.cut(geometry, polyline)</code>.</li>
```

Comment:
I would added to the second line.
Loop through the <code>List&lt;Geometry&gt;</code> of cut geometry parts.